website/frontend/src/pages/ExploreData/index.js

```diff
@@ -13,21 +13,20 @@ import ManDownloadingApp from 'assets/img/explore/man-download-app.png';
 import ManExploring from 'assets/img/explore/get-started-explore.png';
 import RegistrationCompleteSvg from 'assets/svg/explore/registration_complete_svg.svg';
 import { useDispatch } from 'react-redux';
-import {
-  Checkbox, FormControl, FormControlLabel, CircularProgress, Modal,
-} from '@mui/material';
+import { Checkbox, FormControl, FormControlLabel, CircularProgress, Modal } from '@mui/material';
 import { useExploreData } from 'reduxStore/ExploreData/selectors';
 import { postExploreDataRequest, updateExploreData } from 'reduxStore/ExploreData/operations';
 import { isEmpty, isEqual } from 'underscore';
 import { useInitScrollTop } from 'utils/customHooks';
 import { NETMANAGER_URL } from 'config/urls';
 import SEO from 'utils/seo';
+import UserManual from 'assets/docs/AirQo Analytics Platform User Guide.pdf';
```

Comment:
@Baalmart 
The considerations were already considered :)
The document has a version number and date, the file name allows for a proper name for download, the storage of the file should ideally be cloud-based so do we have a stable bucket for that? otherwise this was the agile way of getting work done, the file size is that size due to images within which is fine, and a PR has been made to rectify the deploy preview issue

Reply:
Hi @FaithDaka 
I'm not sure if the concern of filename is for the user side, but it's totally okay for most users to have underscores instead of spaces in the file name. Will let you and @Baalmart decide on how to conclude on this.